style.css

```diff
@@ -41,3 +41,180 @@
 .green {
   color: #36b37e;
 }
+
+.line {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+}
+
+.work-section {
+  width: 100%;
+  align-items: cente;
+  padding: 10px 15px 15px 10px;
+}
+
+.project-container {
+  display: grid;
+  grid-template-columns: 80%;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 20px;
+}
+
+.project {
+  border: 1px solid green;
+  margin: 0;
+  width: 100%;
+  padding: 0;
+  border-radius: 10px;
+}
+
+.recent {
+  text-align: center;
+  font-size: 50px;
+}
+
+.image-placeholder {
+  font-family: 'Inter', sans-serif;
+  font-style: 'semi-bold', italic;
+  font-size: 24px;
+  line-height: 32px;
+  line-height: 110%;
+  text-align: center;
+  width: 100%;
+  height: 200px;
+  background-color: #d0d9d4;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+
+.btn {
+  background-color: #36b37f;
+  color: white;
+  width: 122px;
+  height: 48px;
+  margin: 20px 0;
+  font-size: 17px;
+  border: none;
+  border-radius: 5px;
+}
+
+.language {
+  list-style: none;
+  display: flex;
+  border: 1px solid #ebf0ee;
+  padding: 5px;
+  background-color: #ebf0ee;
+  margin: 0;
+}
+
+.lang {
+  display: flex;
+  justify-content: space-between;
+}
+
+.multi {
+  text-align: center;
+  font-size: 25px;
+}
+
+.butt {
+  text-align: center;
+}
+
+.dis {
+  display: flex;
+  justify-content: center;
+}
+
+.about {
+  text-align: center;
+  font-size: 30px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.about2 {
+  text-align: center;
+  align-items: center;
+  padding: 15px 20px;
+  font-size: 23px;
+}
```

Comment:
Figma design

![Screenshot from 2022-09-23 13-38-57](https://user-images.githubusercontent.com/65233860/191973511-b4b8864e-bc4a-43b4-ad86-4f28f3e3e8a4.png)

As shown in the figma image above, there are differences in the dimensions between your design and that of the figma designs. hence the disparity in the image below (your design). Kindly try as much as possible to stick to the figma dimensions. Check the font-family,  font-style, font-weight, font-size, line-height etc. 

Your design is also missing the background image. Kindly make sure that they are all working and as close to the figma design as possible.

Your design

![Screenshot from 2022-09-23 13-48-11](https://user-images.githubusercontent.com/65233860/191975353-4ec871b7-1ae3-4d66-8658-f527784cf9cf.png)